### ISSUE TITLE
Update information in Lesson Pipeline Page

### ISSUE THREAD

**Issue by jerielizabeth** [programminghistorian/jekyll#243]
In preparation for (potentially) moving the pipeline to the main website pages, we should move everything  from sorted by editor to sorted by "stages" and confirm that the information is up to date.

https://github.com/programminghistorian/jekyll/wiki/Lesson-Pipeline

**Comment by amandavisconti** [comment]
The item under my name (Amanda Visconti) under "First draft returned" can be removed—the lesson is now published.

**Comment by acrymble** [comment]
Thanks @amandavisconti I've removed you now.

**Comment by ianmilligan1** [comment]
Thanks for the reminder @jerielizabeth – I have a few updates to make. Apologies.. have been on the road extensively, but fear not, stuff's going on! 😀

**Comment by acrymble** [comment]
Does this page still serve a purpose? We don't update it regularly, and if someone is REALLY writing a lesson it should have a ticket on the submission's page. The only reason I can think of for keeping it is if someone gets hit by a bus so that we know who (in theory) they were working with.

**Comment by ianmilligan1** [comment]
To be honest, I haven't been the best at keeping it updated for the reasons you outline..

**Comment by wcaleb** [comment]
We discussed this in #261 and decided that the pipeline page does serve two purposes: (a) advertising to potential authors the kinds of lessons we are interested in, something discussed on #152 and (b) preventing duplication of effort with multiple authors working on the same topic. However, we decided that it's not super important to keep track of _where_ a lesson is in the pipeline: the only distinction that really matters is whether it _has_ been submitted to `ph-submissions` (and thus has an issue ticket that will track its progress already) or _hasn't_ (and thus might need occasional prodding from an editor to see if the author still plans to submit). 

We could also set up a convention of going to the Wiki on the day of each monthly meeting and making an update then by removing any lessons that have appeared on the `ph-submissions` site from the pipeline list.

If everyone is amenable to this, then we will restructure the wiki page so that it is just a long list of potentially coming lessons, without authors and editors, and without sections or dates attached that have to be continuously updated.
